LeGO-LOAM/src/imageProjection.cpp

```diff
@@ -27,7 +27,7 @@
 // POSSIBILITY OF SUCH DAMAGE.
 
 #include "utility.h"
-
+#include <array>
```

Comment:
I think after this MR we should agree on a C++ style. Keeping internal package includes, STD, and others close will make the code less readable in the future.

Reply:
array is a c++11 library (standard already used by the library)

It is not less readable then a std::vector, in my opinion. Actually it is more readable to me, because we clearly say to the reader that it is a 4 elements array.

Reply:
Thanks for the feedback.

I mean that we should define the style guidelines for includes.